src/wp-includes/deprecated.php

```diff
@@ -3957,3 +3957,15 @@ function wp_make_content_images_responsive( $content ) {
 	// This will also add the `loading` attribute to `img` tags, if enabled.
 	return wp_filter_content_tags( $content );
 }
+
+/**
+ * Turn register globals off.
+ *
+ * @since WP-2.1.0
+ * @access private
+ * @deprecated WP-5.5.0
+ */
+function wp_unregister_GLOBALS() {  // phpcs:ignore WordPress.NamingConventions.ValidFunctionName.FunctionNameInvalid
+	// register_globals was deprecated in PHP 5.3 and removed entirely in PHP 5.4.
+	_deprecated_function( __FUNCTION__, 'WP-5.5.0' );
```

Comment:
Since we have the deprecation notes in the docblock, shouldn't the the`WP-5.5.0` on line 3970 reflect the CP Version?

Reply:
If deprecated upstream we should use the upstream version. If we deprecate then we should use the CP version.